chat-plugins/info.js

```diff
@@ -12,10 +12,7 @@
 
 'use strict';
 
-/** @typedef {(this: CommandContext, target: string, room: BasicChatRoom, user: User, connection: Connection, cmd: string, message: string) => (void)} ChatHandler */
-/** @typedef {{[k: string]: ChatHandler | string | true | string[]}} ChatCommands */
-
-/** @type {ChatCommands} */
+/** @type {import('./../chat').ChatCommands} */
```

Comment:
!!!!!!!!!!!!!!!

THIS WORKS NOW??????

Reply:
Yeah :D

Reply:
I wonder if we can globalize this, then? Can `import('./../chat').ChatCommands` be stuck in `globals.ts` or something?

Reply:
That's a good idea, I'll try it.

Reply:
Updated with that.